### DISCUSSION TITLE
tailwind modal - deactive close outside click in modals

### DISCUSSION THREAD

**Discussion by yokadance** [tailwindlabs/tailwindcss#4279] [Help]
Hi, im implement modal with taiwind, and i want that when the user click outside the modal, not close it.

My code:

[ModalJS.txt](https://github.com/tailwindlabs/tailwindcss/files/6445124/ModalJS.txt)

tks in advance!

**Comment by elamje** [comment]
@adamwathan Is it possible to change modal behavior for clicks so that the modal doesn't close? Our frontend team is wondering if we need to find another solution for modals. We love tailwind btw!

**Comment by Edelson471** [comment]
Thanks for the info I will try to figure it out for more.

[paycheckrecords](https://www.paycheckrecords.me/)
